app/views/openid_connect/shared/redirect_js.html.erb

```diff
@@ -16,10 +19,10 @@
             <%= t('saml_idp.shared.saml_post_binding.no_js') %>
           </p>
 
-          <%= link_to(t('forms.buttons.continue'), @oidc_redirect_uri, class: 'usa-button usa-button--wide usa-button--big', id: 'openid-connect-redirect') %>
+          <%= link_to(t('forms.buttons.continue'), @oidc_redirect_uri, class: 'usa-button usa-button--wide usa-button--big', data: { click_immediate: '' }) %>
```

Comment:
the SAML one has `click_immediate: true` which personally seems clearer to me than empty string, should we have them match?

```suggestion
          <%= link_to(t('forms.buttons.continue'), @oidc_redirect_uri, class: 'usa-button usa-button--wide usa-button--big', data: { click_immediate: true }) %>
```

Reply:
We should have them match, yeah. I might have a soft preference for empty string since IIRC it's the difference of `<foo data-click-immediate="true" />` vs. `<foo data-click-immediate>`, which doesn't have an effective difference in how it's handled, but I like thinking of this as a [boolean attribute](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML), for which (ironically) `"true"` is invalid.

Reply:
as long as they match I'm happy

Reply:
Using consistent attribute value as of 3594113.